doc/userguide/rules/dcerpc-keywords.rst

```diff
@@ -0,0 +1,67 @@
+DCE/RPC
+=======
+Suricata supports decoding `DCE/RPC <https://en.wikipedia.org/wiki/DCE/RPC>`_ application traffic via the dcerpc application layer protocol. Because of this we are able to detect within DCE/RPC traffic for interface UUIDs, operation numbers, and for bytes within the stub data. This is achieved by using two keywords and a sticky buffer: ``dcerpc.iface``, ``dcerpc.opnum``, and the ``dcerpc.stub_data`` sticky buffer.
+
+
+dcerpc.iface 
+------------
+It is necessary for a DCE/RPC client to bind to a service before being able call to it. When a client sends a bind request to the server it can specify one or more interfaces to bind to. Each interface is represented by a UUID. Each interface UUID is paired with a unique index (or context id) that future requests can use to reference the service that the client is making a call to. Using the ``dcerpc.iface`` rule option, we can determine whether or not the client has bound to a specific interface UUID and whether or not subsequent client requests are making a request to it. An interface contains a version, and some versions of an interface may not be vulnerable to a certain exploit. We can specify a version or range of versions by setting the ``version`` option with operators. Also, a DCE/RPC request can be broken up into 1 or more fragments. Flags are set in the DCE/RPC header to indicate whether the fragment is the first, a middle or the last fragment. Some checks for data in the DCE/RPC request are only relevant if the DCE/RPC request is a first fragment since subsequent fragments will contain data deeper into the DCE/RPC request. By default it is reasonable to only evaluate if the request is a first fragment. However, if the ``any_frag`` option is used to specify evaluating on all fragments.
```

Comment:
Can you split this up to keep it in line with the ~80 char line limit. The html/pdf output won't be affected by it.